common/spec/dependabot/pull_request_creator/message_builder/metadata_presenter_spec.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "dependabot/pull_request_creator/message_builder/metadata_presenter"
+
+namespace = Dependabot::PullRequestCreator::MessageBuilder
+RSpec.describe namespace::MetadataPresenter do
+  let(:source) do
+    Dependabot::Source.new(provider: "github", repo: "gocardless/bump")
+  end
+
+  let(:dependency) do
+    Dependabot::Dependency.new(
+      name: "business",
+      version: "1.5.0",
+      previous_version: "1.4.0",
+      package_manager: "dummy",
+      requirements:
+        [{ file: "Gemfile", requirement: "~> 1.5.0", groups: [], source: nil }],
+      previous_requirements:
+        [{ file: "Gemfile", requirement: "~> 1.4.0", groups: [], source: nil }]
+    )
+  end
+
+  let(:credentials) do
```

Comment:
Question: are these used?

Reply:
Opps, they aren't - in a previous version I used a real `MetadataFinder::Base`